frappe_whatsapp/setup/install.py

```diff
@@ -0,0 +1,61 @@
+import frappe
+
+def create_integrations_card():
+    try:
+        # Fetch the Integrations workspace document
+        workspace = frappe.get_doc('Workspace', 'Integrations')
+    except frappe.DoesNotExistError:
+        frappe.throw("The Integrations workspace does not exist.")
```

Comment:
Instead of throwing an error, we can create an integration workspace.

Reply:
The Integration Workspace is standard in Frappe, this error should not occur excpet in a case where frappe is not yet installed.

Reply:
Throwing an error will not cause any issue in installing app?

Reply:
The installation still continues, the only thing that happens is that it won't create the card in the Integrations Workspace. I try to retain Frappe best practices with my apps and the exit routine (Error) was built in to escape the setup.py script without breaking the installation (it runs afterinstall, it could also be run aftermigrate, I think aftermigrate would have been better though, but we can fix that in the hooks.py if needed).

Reply:
yeah I think it should be after migrate as this will not create card if app is already installed.

Reply:
We can then edit out after_install ... and replace with
after_migrate = "frappe_whatsapp.setup.install.create_integrations_card"